src/helpers/figures/charts/scorecard_chart_config_builder.ts

```diff
@@ -50,8 +46,19 @@ export type ScorecardChartConfig = {
     position: PixelPosition;
   };
   baseline?: ScorecardChartElement;
-  baselineDescr?: ScorecardChartElement;
```

Comment:
could it be just changed to an array of elements ? `baselineDescr?: ScorecardChartElement[];`
There would be less ifs in the scorecard component such as `if (structure.baselineDescr.splitted)`)

The benefit would be to keep all the "complex" computations on the model/plugin side rather than in the component (because components are hard to test, even more when it's actually a canvas)